fe/fe-core/src/main/java/com/starrocks/planner/HdfsScanNode.java

```diff
@@ -93,9 +93,11 @@ private void setupCloudCredential() {
             return;
         }
         CatalogConnector connector = GlobalStateMgr.getCurrentState().getConnectorMgr().getConnector(catalog);
-        if (connector != null) {
-            cloudConfiguration = connector.getCloudConfiguration();
-        }
+        Preconditions.checkState(connector != null,
```

Comment:
extract a common static function in ScanNode?

Reply:
Dear, let's refactor that in next PR.